apps/mutations/migrations/0001_initial.py

```diff
@@ -46,10 +46,10 @@ class Migration(migrations.Migration):
         ),
         migrations.AlterUniqueTogether(
             name='mutation',
-            unique_together=set([('gene_locus', 'name')]),
+            unique_together={('gene_locus', 'name')},
         ),
         migrations.AlterUniqueTogether(
             name='genelocus',
-            unique_together=set([('drug', 'name')]),
```

Comment:
See above, changing auto-generated files.

Reply:
I can revert this change, but this would not affect migrations. Literal is much faster since it uses optimized BUILD_MAP and STORE_MAP opcodes rather than generic CALL_FUNCTION.

```bash
> python3.2 -m timeit "d = set('name', 'country')"
1000000 loops, best of 3: 0.975 usec per loop

> python3.2 -m timeit "s = {'name', 'country'}"
1000000 loops, best of 3: 0.409 usec per loop
```